src/utilities/grid/Grid.mixins.ts

```diff
@@ -0,0 +1,41 @@
+import { css } from 'styled-components';
+import { FlattenSimpleInterpolation } from 'styled-components/ts3.6';
+
+import { Utils } from '../../shared';
+
+import { PApplyGrid, PApplyGridItem } from './Grid.props';
+
+const applyGrid = ({
+    columnsTemplate = 'none',
```

Comment:
I feel like it would be a good user experience if we add a comment on what these properties do, I like to do this with functions with many params and especially when its applying styles.

Reply:
done